frontend/components/receipts/transfer/index.tsx

```diff
@@ -1 +1,79 @@
-export * from './transfer'
+import ReactTimeAgo from 'react-time-ago'
+import { useEffect, useState } from 'react'
+import { isBefore } from 'date-fns'
+import { ReceiptComponentProps } from '../receipts'
+import { Transaction } from '../../../lib/transactions'
+import { BasicTransferView } from './basic-transfer-view.tsx'
+import { EnrichedTransferView } from './enriched-transfer-view.tsx'
+import { getDateTimeFormat } from '@vegaprotocol/utils'
+import { ReceiptWrapper } from '../utils/receipt-wrapper'
+import { useAssetsStore } from '../../../stores/assets-store.ts'
+import { useWalletStore } from '../../../stores/wallets.ts'
+import { VegaAsset } from '../../../types/rest-api.ts'
+
+const getTime = (transaction: Transaction) => {
+  const deliverOn = transaction.transfer.oneOff?.deliverOn
+  if (deliverOn) {
+    const deliverOnInSeconds = Math.floor(deliverOn / 1000)
```

Comment:
Nano to milli is 10^6 no?

```suggestion
    const deliverOnInSeconds = Math.floor(deliverOn / 1e6)
```